.github/workflows/test.yml

```diff
@@ -136,6 +136,32 @@ jobs:
           flags: unittests
     needs: setup
 
+  test-types:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [12]
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: checkout
+        uses: actions/checkout@master
+
+      - name: restore workspace cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ github.workspace }}
+          key: ${{ matrix.os }}-node-v${{ matrix.node }}-nuxt-${{ github.sha }}
+
+      - name: test types
```

Comment:
What do you think to simply add step to lint workflow to avoid extra setup?

Reply:
done :)